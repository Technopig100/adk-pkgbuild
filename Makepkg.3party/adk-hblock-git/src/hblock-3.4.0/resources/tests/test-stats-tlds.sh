#!/bin/sh

# Author:     Héctor Molinero Fernández <hector@molinero.dev>
# License:    MIT, https://opensource.org/licenses/MIT
# Repository: https://github.com/hectorm/hblock

set -eu
export LC_ALL='C'

SCRIPT_DIR="$(CDPATH='' cd -- "$(dirname -- "${0:?}")" && pwd -P)"

# shellcheck disable=SC1091
. "${SCRIPT_DIR:?}"/env.sh

main() {
	printf 'Test - Stats: TLDs\n'
	actual="$(runInTestShell "${SCRIPT_DIR:?}/../stats/stats.sh" "${SCRIPT_DIR:?}/test-domains-stats.txt" none)"
	expected="$(cat -- "${0%.sh}".out)"
	if ! assertEquals "${actual?}" "${expected?}"; then
		exit 1
	fi
}

main "${@-}"
