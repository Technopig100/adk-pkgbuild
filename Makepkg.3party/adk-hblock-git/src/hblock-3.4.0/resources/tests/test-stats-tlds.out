Top	Hosts	Suffix
1	12	.xn--j6w193g
2	9	.arpa
3	9	.jp
4	4	.com
5	4	.net
6	3	.org
7	1	.unknown
